src/evergreen/__init__.py

```diff
@@ -1,5 +1,21 @@
 """Evergreen API Module."""
 from pylibversion import version_tuple_to_str
 
-VERSION = (0, 6, 12)
+VERSION = (0, 6, 13)
 __version__ = version_tuple_to_str(VERSION)
+
+
+# Shortcuts for importing.
```

Comment:
This seems like common practice in python. It should make importing the project a lot easier.

Reply:
Yes, it allows importing everything from the evergreen module:
```
from evergreen import EvergreenApi, Version, Patch
```